lib/stackprof/report.rb

```diff
@@ -9,7 +9,7 @@ def initialize(data)
     attr_reader :data
 
     def frames(sort_by_total=false)
-      Hash[ *@data[:frames].sort_by{ |iseq, stats| -stats[sort_by_total ? :total_samples : :samples] }.flatten(1) ]
+      *@data[:frames].sort_by{ |iseq, stats| -stats[sort_by_total ? :total_samples : :samples] }.inject({}){|h, (k, v)| h[k] = v; h}
```

Comment:
Doesn't this cause a syntax error?
```ruby
2.2.3 :001 > def a
2.2.3 :002?>   *[1, 2]
2.2.3 :003?> end
SyntaxError: (irb):1: syntax error, unexpected '\n', expecting :: or '[' or '.'
	from ~/.rvm/rubies/ruby-2.2.3/bin/irb:15:in `<main>'
```

It appears that stackprof isn't set up to run its tests on a CI service. @tmm1 is that something you'd want? I'd open a PR myself, except I'm pretty sure only people with commit access can configure Travis and the like.